rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm

```diff
@@ -85,6 +85,31 @@
 				${rlb.getFormattedMessage("jsp_last_modified_by", $value.toArray())}
 				<a id="toCommentsLink" href="#tocomments" style="display:none" style="padding:0 1em 0 0">${rlb.jsp_comment_gotolink} <span></span></a>
 			 </div>
+			#if($rwikiObjectGroups)
+				<div class="groupsforthispage specialLink" >
+					${rlb.view_groups_text}
+					<span class="bold-text">
+						#set ($isFirst = true)
+						#set($count = 0)
+						#foreach($group in $groups)
+							#foreach($rwikiObjectGroup in $rwikiObjectGroups)
```

Comment:
Can't you use one of the already formed strings that are on the other files?

Reply:
I don't think so, because i use the rwikiObjectGroups (that are the groupsIds) and the groups (that are the all groups object in the rwiki pages)